### PR TITLE
A commit to create a Pull Request

### PR DESCRIPTION
**Pull request by jdcasi**
doing changes to test PR

### CODE REVIEW

**Comment by ben-tuttle** [comment]
Angry blocky birds

File: index.ios.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const styles = StyleSheet.create({
   },
   etherRemaining: {
     paddingHorizontal: 15,
-    paddingTop: 8,
+    paddingTop: 7,
```
Comment (ben-tuttle) [inline index.ios.js]: This is ridiculous!! How could anyone ever use anything with paddingTop: 7!!!!!!!

Comment (ben-tuttle) [inline index.ios.js]: hats

Comment (jdcasi) [inline index.ios.js]: ok i got your fix

**Comment by tfahy** [comment]
Can you please remove me from this thread?  Thanks

**Comment by wesholleradc** [comment]
lol. not good enough jcasi.

File: index.ios.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Sample React Native App
+ * Hello World React Native App
```
Comment (wesholleradc) [inline index.ios.js]: False dichotomy between `Sample` and `Hello World`.

**Comment by jdcasi** [comment]
@wesholleradc please re-review, currently if you request changes, then this tool requires you to look at changes and approve them.  Basically if you dip your toe in the review water, you are in the pool until you give approval.

**Pull request closed without merging**